### PR TITLE
steps/masters: add libvirt support

### PR DESCRIPTION
**Pull request by squeed**
Adds libvirt support for creating multiple masters.

### CODE REVIEW

**Comment by enxebre** [comment]
looks good! thanks for changing the variable name, let's run the aws smoke tests

**Comment by enxebre** [comment]
ok to test

**Comment by squeed** [comment]
tests are greener than a 5€ bill.

File: steps/joining_workers/aws/workers.tf [hunk 1]
```diff
@@ -10,14 +10,14 @@ provider "aws" {
 }
 
 module "container_linux" {
-  source = "../../modules/container_linux"
+  source = "../../../modules/container_linux"
```
Comment (enxebre) [inline steps/joining_workers/aws/workers.tf]: we should probably move this path to a local variable at some point

**Pull request merged**